src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/integration_tests/tests/web/WorkflowRunTests.kt

```diff
@@ -149,11 +149,7 @@ class WorkflowRunTests : IntegrationTest()
                       }
                     },
                     {
-                      "name": "minimal",
-                      "params": {}
-                    },
-                    {
-                      "name": "global",
+                      "name": "connection",
```

Comment:
This test was failing, possibly because the orderly demo has changed, and minimal and global reports not in 'other' branch.

Reply:
Sounds totally likely but did this not show up in your [test branch](https://github.com/vimc/orderly-web/pull/401)?

Reply:
No, it didn't actually, I tried reverting this locally, and it passed this time! But those reports certainly don't exist on the 'other' branch, so I wouldn't expect it to. I'll try running it through BK again, and will check with Rob what he thinks might be going on.

Reply:
Talked to Rob, and I think we've got to the bottom of this  - there's actually a bit of a bug in the report lists we're getting back. The endpoint we call on orderly server, `reports/source` actually only returns reports which are not in master or which have changes from master, for non-master branches. This was done when creating the feature for running a single report, to avoid cluttering up the drop-down with irrelevant reports when testing a branch (probably with a single new report in it). But it's perfectly fine to run those unchanged reports. So in the case of the git demo, minimal and global have not been deleted from 'other' branch as we thought - they just don't have changes so are not returned in the list. Hence this test  workflow should indeed run successfully. 

Obviously this isn't going to be desirable when creating workflows as we may need to create dependencies which are in master - Rob is to going to add a flag to the orderly server endpoint to return unchanged report too, and we can then use this when getting report lists for workflows. (This will also probably break some of our other tests which assume global ad minimal don't exist in 'other'.)

Ticket for using new flag once it exists: https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-2829